src/commands/test.rs

```diff
@@ -350,6 +418,20 @@ pub async fn run_submission_tests(
                     score
                 ));
             }
+            pipeline_pb.println(format!(
+                "{}: {}",
+                "Score".if_supports_color(OwoStream::Stdout, |text| { text.bold() }),
+                score
+            ));
+            if let Some(path) = project_readme_path {
+                update_score_badge_in_file(
+                    global.project.join(path),
+                    modified_use_case.competition,
+                    &score,
+                    global.aqora_url()?,
+                )
+                .await?;
```

Comment:
I would maybe just emit a warning here if we can't write to the README as it would be 
a shame to throw an error for something that's decorative and make people feel like they need to rerun their tests